src/main/stackpack/provisioning/SplunkTopologyProvision.groovy

```diff
@@ -11,18 +11,17 @@ class SplunkTopologyProvision extends ProvisioningScript {
 
   @Override
   ProvisioningIO<scala.Unit> install(Map<String, Object> config) {
-    def cleanConfig = cleanupConfig(config)
-
+    def instance_name = config.splunk_instance_name.trim()
+    def args = ["instanceName": instance_name]
     def templateArguments = [
-      'topicName': topicName(cleanConfig),
-      'integrationType': cleanConfig.sts_instance_type,
-      'integrationUrl': integrationUrl(cleanConfig),
-      'instanceId': context().instance().id()
+            'topicName'   : topicName(config),
+            'instanceName': instance_name,
+            'instanceId'  : context().instance().id()
     ]
-    templateArguments.putAll(cleanConfig)
+    templateArguments.putAll(config)
 
-    return context().stackPack().importSnapshot("templates/splunk-topology-template.stj", [:]) >>
-           context().instance().importSnapshot("templates/splunk-topology-instance-template.stj", templateArguments)
+    return context().stackPack().importSnapshot("templates/splunk-topology-template.stj", args) >>
```

Comment:
We shouldn't be needing `instanceName` here, `instanceName` is put in `args`. No arguments, [:], should be enough for the shared template, I would assume.

Reply:
Also, the variable is unused in the template file.

Reply:
correct we don't need `args` here and moved that now.